nessie/lakectl_test.go

```diff
@@ -295,3 +301,52 @@ func TestLakectlAuthUsers(t *testing.T) {
 	// Cleanup
 	RunCmdAndVerifySuccess(t, Lakectl()+" auth users delete --id "+userName, false, "User deleted successfully\n", vars)
 }
+
+func TestLakectlIngest(t *testing.T) {
```

Comment:
This test can't work with Azure and GCP.

Reply:
Correct
Added the adapter type to nessie via env var. Test is skipped if adapter is not S3